external-crates/move/crates/move-compiler/src/shared/ide.rs

```diff
@@ -49,11 +49,11 @@ pub struct MacroCallInfo {
 
 #[derive(Debug, Clone)]
 pub struct AutocompleteInfo {
-    /// Methods that are valid autocompletes
-    pub methods: BTreeSet<(E::ModuleIdent, P::FunctionName)>,
-    /// Fields that are valid autocompletes (e.g., for a struct)
-    /// TODO: possibly extend this with type information?
-    pub fields: BTreeSet<Symbol>,
+    /// Methods that are valid autocompletes (FunctionName represents target function and Symbol
+    /// represents method name)
+    pub methods: BTreeSet<((E::ModuleIdent, P::FunctionName), Symbol)>,
```

Comment:
Consider defining a struct to hold these values, as it will make the code easier to reason about.
```
struct AutocompleteMethod {
    method_name: Symbol,
    target_function: (E::ModuleIdent, P::FunctionName)
}
```